applications/GeoMechanicsApplication/custom_processes/apply_scalar_constraints_table_process.h

```diff
@@ -0,0 +1,63 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf,
+//                   Marjan Fathian
+//
+#pragma once
+
+#include "processes/process.h"
```

Comment:
I think here Process can also be forward declared (like ModelPart and Parameters)